src/ameba/rule/naming/block_parameter_name.cr

```diff
@@ -0,0 +1,56 @@
+module Ameba::Rule::Naming
+  # A rule that reports non-descriptive block parameter names.
+  #
+  # Favour this:
+  #
+  # ```
+  # tokens.each { |token| token.last_accessed_at = Time.utc }
+  # ```
+  #
+  # Over this:
+  #
+  # ```
+  # tokens.each { |t| t.last_accessed_at = Time.utc }
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Naming/BlockParameterName:
+  #   Enabled: true
+  #   MinNameLength: 1
+  #   AllowNamesEndingInNumbers: true
+  #   AllowedNames: [_, e, i, j, k, v, x, y, ex, io, ws, tx, id, k1, k2, v1, v2]
+  #   ForbiddenNames: []
+  # ```
+  class BlockParameterName < Base
+    include AST::Util
+
+    properties do
+      description "Disallows non-descriptive block parameter names"
+      min_name_length 1
```

Comment:
just curious if the default value can be increased here to 3. I believe rubocop enforces block args to have a min length of 3, any benefits of this?

Reply:
I agree, I've forgot to change it.

Reply:
Just ftr, Rubocop has default set to `1`.